packages/message/src/types.ts

```diff
@@ -1,15 +1,29 @@
-export type MessageTemplate = {
-  cumulus_meta: object
+import { Message } from '@cumulus/types';
+
+export interface MessageTemplateCumulusMeta {
+  queueExecutionLimits: {
+    [queueUrl: string]: number
+  }
+}
+
+export interface MessageTemplate {
+  cumulus_meta: MessageTemplateCumulusMeta
   meta: object
-};
+}
 
-export type Workflow = {
+export interface Workflow {
   arn: string
   name: string
-};
+}
 
-export type QueueMessageMeta = {
+export interface QueueMessageMeta {
   workflow_name: string
   collection?: object
   provider?: object
-};
+}
+
+export interface CumulusQueueMessage {
```

Comment:
🐜 Consider something like `QueuableCumulusMessage`? That seems to _slightly_ better describe what this thing is.

On a related note, do we want to consider occasionally adding a comment to the code describing where a particular type is intended to be used?